src/OpenTelemetry.Resources.Host/HostDetectorOptions.cs

```diff
@@ -0,0 +1,20 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+namespace OpenTelemetry.Resources.Host;
+
+/// <summary>
+/// Provides options for configuring the host resource detector.
+/// </summary>
+public class HostDetectorOptions
+{
+    /// <summary>
+    /// Gets or sets a value indicating whether the mac ID of the host is to be included as a resource attributes.
+    /// </summary>
+    public bool IncludeMac { get; set; }
+
+    /// <summary>
+    /// Gets or sets a value indicating whether the ip of the host is to be included as a resource attributes.
+    /// </summary>
+    public bool IncludeIp { get; set; }
```

Comment:
I believe `IncludeIP` would be the correct casing by .NET conventions (e.g. [`IPAddress`](https://learn.microsoft.com/dotnet/api/system.net.ipaddress), [`IPEndPoint`](https://learn.microsoft.com/dotnet/api/system.net.ipendpoint)).